### PR TITLE
Added a `signed` option in gyrofrequency

### PR DESCRIPTION
**Pull request by jhillairet**
This PR should close issue #206

### CODE REVIEW

**Comment by jhillairet** [comment]
I don't understand the coverall error... could someone explain me ?

File: plasmapy/physics/parameters.py [hunk 1]
```diff
@@ -790,24 +798,19 @@ def plasma_frequency(n, particle='e'):
...
 
     try:
         m = atomic.ion_mass(particle)
-        try:
-            Z = atomic.charge_state(particle)
-        except ValueError:
-            Z = 1
     except Exception:
-        raise ValueError(f"Invalid particle, {particle}, in "
+        raise ValueError("Invalid particle, {particle}, in "
```
Comment (StanczakDominik) [inline plasmapy/physics/parameters.py]: Keep this as an f string or it will literally print {particle}!

**Comment by StanczakDominik** [comment]
Literally one thing to be fixed up and this can be merged! Thanks for getting it done!

File: plasmapy/physics/parameters.py (hunk 1 above)
Comment (jhillairet) [reply plasmapy/physics/parameters.py]: Indeed ! I didn't knew about this convenient format shorcut, nice !

**Comment by StanczakDominik** [comment]
And done, thank you!

**Pull request merged**